docs/modules/operation/pages/service-assurance/polling-packages.adoc

```diff
@@ -272,10 +271,10 @@ Service is Up on 192.168.31.100 using org.opennms.netmgt.poller.monitors.IcmpMon
 ----
 <1> Service and package of this test
 <2> Applied service configuration from polling package for this test
```

Comment:
It wouldn't let me include all of these in the suggestion, so I apologize for the number of comments in this section.

```suggestion
<1> Service and package of this test.
<2> Applied service configuration from polling package for this test.
```